rott/cin_main.c

```diff
@@ -247,8 +247,9 @@ void R_DrawFilmColumn (byte * buf)
 	int frac, fracstep;
 	byte *dest;
 
-	count = cin_yh - cin_yl;
-	if (count < 0) return;
+	count = (cin_yh + 1) - cin_yl;
```

Comment:
Please use the same semantics as e.g. `R_DrawColumn()`:

https://github.com/fabiangreffrath/taradino/blob/edfa0296b7c4324ca4f0c527b854aa9ac7b40640/rott/rt_scale.c#L1091-L1092

Reply:
Done!